app/Http/Controllers/Api/GeneralController.php

```diff
@@ -44,4 +46,48 @@ public function version()
             'latest'    => $latest,
         ])->item(CACHET_VERSION);
     }
+
+    /**
+     * Show the general system status.
+     *
+     * @return \Illuminate\Http\JsonResponse
+     */
+    public function status()
+    {
+        $totalComponents = Component::enabled()->count();
+        $majorOutages = Component::enabled()->status(4)->count();
```

Comment:
Is this code taken from the other controller? If so, we should probably reuse it, not copy it.